dattri/model_util/retrain.py

```diff
@@ -206,6 +207,9 @@ def train_func(dataloader, seed=None, **kwargs):
         ValueError: If `total_num_subsets` is negative.
         ValueError: If `num_subsets` does not divide `total_num_subsets`.
     """
+    if seed is None:
+        seed = random.getrandbits(64)
```

Comment:
Could you add a in-line comment for this block? Like "generate a random seed if seed is not explicitly provided."